test/e2e/specs/editor/various/convert-block-type.spec.js

```diff
@@ -3,7 +3,7 @@
  */
 const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
 
-test.describe( 'Code block', () => {
+test.describe( 'Block transforms', () => {
```

Comment:
There are existing tests for individual blocks that verify transforms, so maybe it would be better to move them there as there might some duplication created over time:

<img width="455" alt="Screenshot 2024-02-20 at 09 56 01" src="https://github.com/WordPress/gutenberg/assets/699132/c2e393e2-5677-41c6-a57e-0e4c268fb788">